recipe/recipe.go

```diff
@@ -1,35 +1,38 @@
 package recipe
 
+//List - struct for list of recipes
 type List struct {
-	Count        int        `json:"count"`
-	Start        int        `json:"start_index"`
-	ItemsPerPage int        `json:"items_per_page"`
-	Items        []Response `json:"items"`
-	TotalCount   int        `json:"total_count"`
+	Count        int        `bson:"count,omitempty" json:"count,omitempty"`
+	Start        int        `bson:"start_index,omitempty" json:"start_index,omitempty"`
+	ItemsPerPage int        `bson:"items_per_page,omitempty" json:"items_per_page,omitempty"`
+	Items        []Response `bson:"items,omitempty" json:"items,omitempty"`
+	TotalCount   int        `bson:"total_count,omitempty" json:"total_count,omitempty"`
 }
 
+//Response - struct for individual recipe
 type Response struct {
-	ID              string     `json:"id"`
-	Alias           string     `json:"alias"`
-	Format          string     `json:"format"`
-	InputFiles      []file     `json:"files"`
-	OutputInstances []instance `json:"output_instances"`
+	ID              string     `bson:"_id,omitempty" json:"id,omitempty"`
```

Comment:
Document will always have an `id`, not sure why it is omitempty is used?

omitempty should only be applied to fields which are optional